src/App.js

```diff
@@ -9,37 +9,41 @@ class App extends React.Component {
     super(props);
     this.state = {
       userSearch: '',
-      locationData: '',
+      locationData: {},
       renderMap: false,
     };
   };
 
-  getCityInfo = async (e) => {
+  handleSubmit = (e) => {
     e.preventDefault();
-    let userSearch = e;
-    userSearch.toLowerCase();
+    let city = e.target.city.value;
     this.setState({
-      userSearch
+      userSearch: e.target.city.value,
     });
-    let locationData = await axios.get()
-    this.setState({
-      locationData,
-    })
+    this.getCityInfo(city);
   };
 
-  handleSubmit(e) {
-    e.preventDefault();
+  getCityInfo = async (city) => {
+    let url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_LOCATIONIQ_ACCESS_TOKEN}&q=${city}&format=json`;
+    let responseData = await axios.get(url)
     this.setState({
-      renderMap: true,
+      locationData: responseData.data[0]
     })
   };
 
+
   render() {
     return (
       <>
-        <Header title='City Explorers' subtitle='View a map of your favorite city in real time'/>
-        <UserForm getCityInfo={this.getCityInfo} handleSubmit={this.handleSubmit}/>
-        { this.state.renderMap && <img src={this.state.locationData}/> } 
+        <Header title='City Explorers' subtitle='View a map of your favorite city in real time' />
```

Comment:
I haven't used the `title` and `subtitle` in react. I'm curious how they handle formatting & styling relative to other text display elements.